### PR TITLE
hazard_regression: Fix docstring to reflect rv_discrete_float return type

### PR DESCRIPTION
**Pull request by saketkc**
I am not sure if that 'TODO' is still needed?

### CODE REVIEW

**Comment by saketkc** [comment]
Good to merge?

File: statsmodels/duration/hazard_regression.py [hunk 1]
```diff
@@ -1347,26 +1347,20 @@ def get_distribution(self, params):
...
         Parameters
         ----------
-        params : arrayh-like
+        params : array-like
             The model proportional hazards model parameters.
 
         Returns
         -------
-        A list of objects of type scipy.stats.distributions.rv_discrete
+        See statsmodels.duration.hazard_regression.rv_discrete_float
```
Comment (josef-pkt) [inline statsmodels/duration/hazard_regression.py]: this should not be a "see", we don't want to have to look somewhere else to find the explanation for the returns

**Comment by josef-pkt** [comment]
I don't know yet, I'm not familiar enough with this part and will have to look more closely and check the source.
I will get to it today or tomorrow. I'm working down the list of PRs and issues for 0.7

Thanks

**Comment by saketkc** [comment]
@josef-pkt Do you think this can now be merged?

File: statsmodels/duration/hazard_regression.py (hunk 1 above)
Comment (saketkc) [inline statsmodels/duration/hazard_regression.py]: A scipy.stats.distributions.rv_discrete 'like' object?

**Comment by saketkc** [comment]
Closing as insignificant.

**Pull request closed without merging**